src/core/renderController.ts

```diff
@@ -89,23 +84,24 @@ module Plottable {
     export function flush() {
       if (_animationRequested) {
         // Layout
-        _componentsNeedingComputeLayout.values().forEach((component: Component) => component.computeLayout());
+        _componentsNeedingComputeLayout.values().forEach((component) => component.computeLayout());
```

Comment:
Actually I looked around the codebase. We have a mixture anyway. I prefer the typed version in the future, just because TSC enforces it sometimes (for example if you wanted to extract the function into an instance variable somewhere)